README.md

```diff
@@ -242,3 +238,52 @@ Run unit tests for a single test class.
 ```
 vendor/bin/phpunit --filter mod_accredible_xxx_testcase
 ```
+
+### Coding style
+
+This plugin is trying to be consistent and follow the recommendations according to [the Moodle coding style](http://docs.moodle.org/dev/Coding_style).
+
+### Code checker setup
+
+You can check if the code meets the Moodle coding style with PHP_CodeSniffer and Code checker. 
+
+#### 1. PHP_CodeSniffer
+
+Install [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) (PHPCS) if your local machine hasn't installed it yet.
```

Comment:
Better not to mention "local machine" here. This could be just - Install [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) (PHPCS)